edi_oca/models/edi_exchange_record.py

```diff
@@ -38,7 +38,9 @@ class EDIExchangeRecord(models.Model):
         readonly=True,
         model_field="model",
     )
-    exchange_file = fields.Binary(attachment=True)
+    exchange_file = fields.Binary(
+        store=True, compute="_compute_exchange_file", attachment=True
+    )
```

Comment:
These changes break normal behavior. Also, in this way you store 2 or N times the same file for all records in the hierarchy.
We should probably have a field which is not stored which computes the value from the current record or from the parent or not touch the field and rely only on `_get_file_content`. In that case, it should be checked every place where we use the `.exchange_file` directly.

Reply:
Hi ! I don't really see why it broke the normal behavior ? 
Also for the duplication of the file i'm aware, but the other way are way more complex, so i'm not sure what is the best

Reply:
Ok, see next comments